sydnerd/Refactor-Tractor-Whats-Cookin
# Feature/user class refactoring

# Pull Request Template
## Description
* Add test file for user-test
* Update function declarations to arrow functions in user-test
* Update data types in User class to match expected data
* 🚧 Still needs functionality to search ingredient names - this functionality is currently located in the scripts.js file
* 🚧 May need updating to test for casing

## Type of change

## Checklist:
